tests/testStatsdClient.cpp

```diff
@@ -159,13 +140,18 @@ void testSendRecv(uint64_t batchSize, uint64_t sendInterval) {
 
     // Make sure we get the exactly correct output
     if (messages != expected) {
-        std::cerr << "Unexpected stats received by server, got:" << std::endl;
-        for (const auto& message : messages) {
-            std::cerr << message << std::endl;
+        for (size_t i = 0; i < expected.size(); ++i) {
```

Comment:
while i was debugging this trying to get a magical seed that would work for all examples i found changing the formatting of this output makes it easier to debug what is going on